lib/setuid.js

```diff
@@ -0,0 +1,15 @@
+var Config = require("./config");
+
+if (Config.get("setuid.enabled")) {
+    setTimeout(function() {
```

Comment:
I determined the timeout by tring it on my old 1.6ghz single thread laptop. I think 15ms should be long enough to bind ports. if 15ms is long enough on a 10 year old laptop then I think it should be right.